pkg/webhook/resources/storageclass/validator.go

```diff
@@ -138,12 +138,33 @@ func (v *storageClassValidator) validateEncryption(newObj runtime.Object) error
 	newSC := newObj.(*storagev1.StorageClass)
 
 	// Use util.LonghornOptionEncrypted as the key to check if the storage class is encrypted
-	if value, ok := newSC.Parameters[util.LonghornOptionEncrypted]; !ok {
+	if value, ok := newSC.Parameters[util.LonghornOptionEncrypted]; !ok || value != "true" {
```

Comment:
This field can only be boolean in LH volume. We should validate whether the value can be parsed as boolean in webhook.

https://github.com/longhorn/longhorn-manager/blob/3ac6b93c30e9f0839129e6fa3c205e171726340c/k8s/pkg/apis/longhorn/v1beta2/volume.go#L279-L280

Reply:
I switch to using `strconv.ParseBool`.